docs/reference/0-roadmap/supported-versions.md

```diff
@@ -42,14 +40,13 @@ a production deployment of the Ed-Fi Resources API. The Platform includes:
 
 ### Ed-Fi ODS/API
 
-| Version                         | Timeline                             | Data Standard(s)    |
-| ------------------------------- | ------------------------------------ | ------------------- |
-| [6.2.x](/reference/ods-api/6.2) | Active through 2025-2026 school year | 4.0.0               |
-| 7.0.0                           | Replaced by 7.1                      | 4.0.0, 5.0.0        |
-| [7.1.x](/reference/ods-api/7.1) | Active through 2025-2026 school year | 4.0.0, 5.0.0        |
-| [7.2.x](/reference/ods-api/7.2) | ⚠️ inactive - upgrade to 7.3         | 4.0.0, 5.1.0        |
-| [7.3.0](/reference/ods-api/)    | Active through 2025-2026 school year | 4.0.0, 5.2.0        |
-| [7.3.1](/reference/ods-api/)    | Active through 2029-2030 school year | 4.0.0, 5.2.0, 6.0.0 |
+| Version                                       | Timeline                             | Data Standard(s)    |
+| --------------------------------------------- | ------------------------------------ | ------------------- |
+| [6.2.x](/reference/ods-api/6.2)               | Active through 2025-2026 school year | 4.0.0               |
+| 7.0.0                                         | Replaced by 7.1                      | 4.0.0, 5.0.0        |
+| [7.1.x](/reference/ods-api/7.1)               | Active through 2025-2026 school year | 4.0.0, 5.0.0        |
+| [7.2.x](/reference/ods-api/7.2)               | ⚠️ inactive - upgrade to 7.3         | 4.0.0, 5.1.0        |
+| [7.3.x](/reference/ods-api/) (current: 7.3.1) | Active through 2028-2029 school year | 4.0.0, 5.2.0, 6.0.0 |
```

Comment:
Corrected to 2028-2029. Also streamlined to "7.3.x", and did the same for Data Standards 5 and 6 above.